Backend/internal/auth/login.go

```diff
@@ -91,11 +67,11 @@ func LoginHandler(w http.ResponseWriter, r *http.Request) {
 		// will change to FRONTEND_URI soon
 		//_httpUtil.SendResponse(w, "success")
 
-		http.Redirect(w, r, "http://localhost:3000/dashboard", http.StatusMovedPermanently)
+		http.Redirect(w, r, FRONTEND_URI, http.StatusMovedPermanently)
```

Comment:
should this and line 74 redirect to the same location? or should it be something like:
```go
FRONTEND_URL + "/dashboard"
```
and
```go
FRONTEND_URL + "/login"
```